src/utils/platform.ts

```diff
@@ -13,3 +13,13 @@ export const getPlatformShortName = (
   if (platformName.toLowerCase().includes('android')) return 'Android';
   return 'Unspecified';
 };
+
+export const getPlatformAndVersion = (platform: string): string => {
```

Comment:
can you please add a small unit test testing just that function? It would be in `src/__tests__/utils/platform.ts`.
No need to do anything comprehensive, but it would be good to test one android platform string that works (`android-hw-p6-13-0-android-aarch64-shippable-qr`), one that doesn't (`android-5-0-aarch64-release`), and probably one from another OS such as `windows11-64-shippable-qr`.